content/en/docs/setup/install/helm/index.md

```diff
@@ -0,0 +1,236 @@
+
+    {{< text bash >}}
+    $ kubectl create namespace istio-system
+    {{< /text >}}
+
+1. Install the Istio base chart which contains cluster level resources like
+   Custom Resource Definitions (CRDs), cluster role and cluster role bindings
+   and service accounts used by the Istio components:
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istio-base manifests/charts/base
+    {{< /text >}}
+
+1. Install the Istio discovery (istiod) chart which includes the Istio control
+   plane deployment and its associated configuration:
+
+    {{< warning >}}
+    The default chart configuration uses the secure third party tokens for Service
+    Account token projections used by Istio proxies to authenticate with the Istio
+    control plane. Before proceeding to install this chart, you should verify
+    if third party tokens are enabled in your cluster by following the steps
+    describe [here](/docs/ops/best-practices/security/#configure-third-party-service-account-tokens).
+    If third party tokens are not enabled, you should add the option
+    `--set global.jwtPolicy=first-party-jwt` to the following command.
+    If the `jwtPolicy` is not set correctly, the `istiod` pod will not get
+    deployed due to the missing `istio-token` volume.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istiod manifests/charts/istio-control/istio-discovery
+    {{< /text >}}
+
+1. (Optional) Install the Istio ingress gateway chart which contains the ingress
+   gateway components:
+
+    {{< warning >}}
+    Ensure that third part tokens are enabled in your cluster or add `--set
+    global.jwtPolicy=first-party-jwt` to the following command.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istio-ingress manifests/charts/gateways/istio-ingress
+    {{< /text >}}
+
+1. (Optional) Install the Istio egress gateway chart which contains the egress
+   gateway components:
+
+    {{< warning >}}
+    Ensure that third part tokens are enabled in your cluster or add `--set
+    global.jwtPolicy=first-party-jwt` to the following command.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istio-egress manifests/charts/gateways/istio-egress
+    {{< /text >}}
+
+## Verifying the installation
+
+1. Ensure all Kubernetes pods in `istio-system` namespace are deployed and have a
+   `STATUS` of `Running`:
+
+    {{< text bash >}}
+    $ kubectl get pods -n istio-system
+    {{< /text >}}
+
+## Upgrading using Helm
+
+### Migrating from non-Helm installations
+
+If you're migrating from a version of Istio installed using `istioctl` or
+Operator to Helm, you need to delete your current installation and re-install
+Istio using Helm as described above.
+
+{{< warning >}}
+Uninstalling Istio can cause your custom Istio configurations to be lost
+permanently so it is highly recommended to take a backup of your Istio
```

Comment:
Instructing folks how to backup their CR's would be useful iMO. For reusable instructions that mostly work - [For backup](https://github.com/istio/istio.io/pull/3755) and [for restore with some dev attention](https://github.com/istio/istio.io/pull/3756). I think it is more reasonable to provide backup instructions in the event something goes wrong for the customer. Our #0 rule in Istio is no data loss.